src/Tribe/Ticket_Object.php

```diff
@@ -72,6 +72,13 @@ class Tribe__Tickets__Ticket_Object {
 		public $provider_class;
 
 		/**
+		 * Holds the SKU for the ticket
+		 *
+		 * @var string
+		 */
+		public $sku;
```

Comment:
I'm not sure - this was part of a fix from @bordoni so that I could access the value when we pass the ticket via js. I'll test it as `protected` and change if that works.